packages/toggleInputList/components/ToggleInputList.tsx

```diff
@@ -98,7 +98,10 @@ const ToggleInputList = ({
       ? getLabelStyle(hasError)
       : cx(visuallyHidden);
     return (
-      <legend className={cx(legendReset, legendClassName)}>
+      <legend
+        className={cx(legendReset, legendClassName)}
+        hidden={!showListLabel}
```

Comment:
What required the addition of `hidden`? Is it needed in conjunction with this class that applies the visually hidden styles?
```
const legendClassName = showListLabel
      ? getLabelStyle(hasError)
      : cx(visuallyHidden);
```

Reply:
@nataliepina @clintonmedbery according to the current implementation, when we want to hide the label, we use CSS 
```
clip: rect(0 0 0 0)
```
Due to this, elements are still present in the DOM structure and we cannot test 
```
expect(queriedElement).not.toBeInTheDocument();
```
Also, we cannot use 
```
expect(queriedElement).not.toBeVisible();
```
Because according to the react testing library documentation https://github.com/testing-library/jest-dom#tobevisible

> An element is visible if all the following conditions are met:
> 
> - it is present in the document
> - it does not have its css property display set to none
> - it does not have its css property visibility set to either hidden or collapse
> - it does not have its css property opacity set to 0
> - its parent element is also visible (and so on up to the top of the DOM tree)
> - it does not have the [hidden](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/hidden) attribute
> - if it has the open attribute

And according to MDN docs hidden attribute is used when we want to hide a specific part of the DOM without removing it from DOM. So, expecting that is what required here added this hidden flag.